tests/congure_quic/app.config.test

```diff
@@ -0,0 +1,11 @@
+CONFIG_MODULE_CONGURE=y
+CONFIG_MODULE_CONGURE_QUIC=y
+CONFIG_MODULE_CONGURE_TEST=y
+CONFIG_MODULE_FMT=y
+CONFIG_MODULE_SHELL=y
+CONFIG_MODULE_ZTIMER=y
+CONFIG_MODULE_ZTIMER_MSEC=y
+CONFIG_MODULE_ZTIMER_USEC=y
```

Comment:
```suggestion
CONFIG_ZTIMER_USEC=y
```

Due to circular dependency stuff and legacy MODULE use, `MODULE_ZTIMER_USEC` must be hidden and instead we must use `ZTIMER_USEC`...

Reply:
Thanks! However, I just figured, that the test does not need ztimer anyways, so I just removed it here and from the Makefile ;-).

Reply:
Mhh seems like I was mistaken...

Reply:
Good that I took a break... this was not true obviously.. but the ztimer_usec was unnecessary.